backend/question-service/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:latest
+
+WORKDIR /usr/src/app
+
+COPY go.mod go.sum ./
```

Comment:
`go.sum` is in .gitignore

Some blog somewhere said to keep it in VC tho https://golangbyexample.com/go-mod-sum-module/

Fixed by installing go and running `go mod download` to create the `go.sum` files

Reply:
thanks for spotting this. i have removed go.sum from the gitignore